src/deploy/NVA_build/noobaa_init.sh

```diff
@@ -121,7 +121,11 @@ prepare_mongo_pv() {
 }
 
 run_endpoint_container() {
-  /usr/local/bin/node ./src/s3/s3rver_starter.js
+  while true
+  do
+    echo "$(date) Running endpoint container ..."
+    /usr/local/bin/node ./src/s3/s3rver_starter.js
+  done
```

Comment:
OK, I added handling of both the endpoint and the pv-pool agents to check env/file called NOOBAA_INIT_MODE which by default is empty and then the container will exit, but if it has the value of "auto" it will restart automatically, and if "manual" it will hang waiting for manual intervention.

Reply:
Great Alternative